<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monitor</class>
 <widget class="QDialog" name="Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/backgd1.png);</string>
  </property>
  <widget class="QLabel" name="safe_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ff0066;</string>
   </property>
   <property name="text">
    <string>  门禁状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="set_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffff33;</string>
   </property>
   <property name="text">
    <string>   设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="tem_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffff33;</string>
   </property>
   <property name="text">
    <string>   温度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ctl_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffff33;</string>
   </property>
   <property name="text">
    <string>  远程控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffff33;</string>
   </property>
   <property name="text">
    <string>   光照：</string>
   </property>
  </widget>
  <widget class="QLabel" name="hum_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ffff33;</string>
   </property>
   <property name="text">
    <string>   湿度：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>58</width>
     <height>58</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>58</width>
     <height>58</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/set.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ctl_pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/ctl.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/exit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="temp_textBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="light_textBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="hum_textBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="RFID_textBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
